packages/shopify-app-remix/docs/generated/generated_static_pages.json

```diff
@@ -0,0 +1,135 @@
+[
+  {
+    "id": "shopify-app-remix",
+    "title": "Shopify App package for Remix",
```

Comment:
I think what is tripping me up about this currently it feels like the expectation is the reader is following along like a tutorial, instead of each section being more standalone, where the reader is learning about concept possibly in isolation.

Reply:
Similar to the Checkout UI extension [references](https://shopify.dev/docs/api/checkout-ui-extensions)